_data/simple-icons.json

```
@@ -5215,6 +5215,21 @@
             "source": "https://gradle.com/brand",
             "guidelines": "https://gradle.com/brand"
         },
+        {
+            "title": "Gradle Play Publisher",
+            "hex": "82B816",
+            "source": "https://github.com/Triple-T/gradle-play-publisher/blob/master/assets/logo.svg",
+            "license": {
+                "type": "MIT",
+                "url": "https://github.com/Triple-T/gradle-play-publisher/blob/master/LICENSE"
+            },
+            "aliases": {
+                "aka": [
+                    "gpp",
+                    "Triple-T"
+                    ]
```

```suggestion
                ]
```
Failing at linter. This correction should fix the issue. You can run `npm run lint` before comitting to check for any issues.